### ISSUE TITLE
`truncate_table': undefined method `terminal_size' for #<HighLine:0x...> (NoMethodError)

### ISSUE THREAD

**Issue by reedloden** [presidentbeef/brakeman#842]
Installed `brakeman` in a new environment today, and when I tried to demo it to somebody, I got the following error:

```
| => brakeman
...
Generating report...
/usr/local/var/rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/brakeman-3.2.1/lib/brakeman/util.rb:446:in `truncate_table': undefined method `terminal_size' for #<HighLine:0x007fb0cdbcfb80> (NoMethodError)
	from /usr/local/var/rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/brakeman-3.2.1/lib/brakeman/report/report_table.rb:7:in `generate_report'
	from /usr/local/var/rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/brakeman-3.2.1/lib/brakeman/report.rb:76:in `generate'
	from /usr/local/var/rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/brakeman-3.2.1/lib/brakeman/report.rb:67:in `to_s'
	from /usr/local/var/rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/brakeman-3.2.1/lib/brakeman/report.rb:38:in `format'
	from /usr/local/var/rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/brakeman-3.2.1/lib/brakeman.rb:373:in `block in write_report_to_formats'
	from /usr/local/var/rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/brakeman-3.2.1/lib/brakeman.rb:372:in `each'
	from /usr/local/var/rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/brakeman-3.2.1/lib/brakeman.rb:372:in `write_report_to_formats'
	from /usr/local/var/rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/brakeman-3.2.1/lib/brakeman.rb:355:in `scan'
	from /usr/local/var/rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/brakeman-3.2.1/lib/brakeman.rb:61:in `run'
	from /usr/local/var/rbenv/versions/2.1.8/lib/ruby/gems/2.1.0/gems/brakeman-3.2.1/bin/brakeman:79:in `<top (required)>'
	from /usr/local/var/rbenv/versions/2.1.8/bin/brakeman:23:in `load'
	from /usr/local/var/rbenv/versions/2.1.8/bin/brakeman:23:in `<main>'
```

When I installed `brakeman`, the following other gems were installed:
- sexp_processor-4.7.0
- ruby_parser-3.8.1
- ruby2ruby-2.3.0
- terminal-table-1.5.2
- highline-2.0.0.pre.develop.6
- haml-4.1.0.beta.1
- temple-0.7.6
- slim-3.0.6
- brakeman-3.2.1

Looks like `highline` may have made some change that `brakeman` is relying on? In the gemspec, `s.add_dependency "highline", ">=1.6.20", "<2.0"` is there, but since `2.0.0.pre.develop.6` is less than `2.0`, it still gets installed.

**Comment by presidentbeef** [comment]
Hi Reed,

I don't know why this happens (see #767), but Brakeman's dependencies are declared properly. If I `gem install highline` I get 1.7.8 and if I `gem install brakeman` I also get 1.7.8.

How did you install Brakeman? What version of `gem` do you have?

**Comment by reedloden** [comment]
I did `gem install brakeman`. Output of `gem -v` is `2.2.5`.

Sorry I missed #767. My github search skills failed, apparently.

**Comment by presidentbeef** [comment]
Try upgrading rubygems: `gem update --system` 2.2.x is pretty old, although 2.2.5 is a newer update for a security issue.

A *ton* of bugs were fixed in 2.3, including (I believe) this one per https://github.com/rubygems/rubygems/blob/master/History.txt#L532-L534

**Comment by reedloden** [comment]
Yup, that solved it. Thanks!
